rakefile

```diff
@@ -102,19 +102,19 @@ rototiller_task :setup_recording_agent do
   ENV['ABS_RESOURCE_HOSTS'] = File.open("last_abs_resource_hosts.log").readlines[0] if File.exist? "last_abs_resource_hosts.log"
   ENV['BEAKER_PRE_SUITE'] = "setup/install_gatling/00_pre_install/05_initialize_helpers.rb,setup/install_gatling/setup_metrics_as_agent.rb"
   ENV['BEAKER_TESTS'] = ""
-  Rake::Task["performance_without_provision"].execute
+  Rake::Task["performance_without_provision"].invoke
 end
 
 desc 'Run Performance tests using previously provisioned hosts'
 rototiller_task :performance_against_already_provisioned do
   ENV['BEAKER_HOSTS'] ||= "log/latest/hosts_preserved.yml"
   ENV['ABS_RESOURCE_HOSTS'] = File.open("last_abs_resource_hosts.log").readlines[0] if File.exist? "last_abs_resource_hosts.log"
   ENV['BEAKER_PRE_SUITE'] = ""
-  Rake::Task["performance_without_provision"].execute
+  Rake::Task["performance_without_provision"].invoke
 end
 
 desc 'Execute beaker performance setup and tests against existing hosts specified by ABS_RESOURCE_HOSTS env var, intended to be used in CI - use task "performance" for local/dev execution'
-rototiller_task :performance_without_provision do |t|
+rototiller_task :performance_without_provision => "validate:validate_performance_env_vars" do |t|
```

Comment:
If I run the performance task it depends on validate_performance_env_vars and then calls this task... will it do teh validation twice?

Reply:
Good catch; I updated `performance` to call `performance_without_provision` using `invoke` which should prevent the validation from being performed twice.